rust/feature-flags/src/flag_request.rs

```diff
@@ -160,6 +160,7 @@ impl FlagRequest {
         redis_client: Arc<dyn RedisClient + Send + Sync>,
         pg_client: Arc<dyn DatabaseClient + Send + Sync>,
     ) -> Result<FeatureFlagList, FlagError> {
+        // TODO add a cache hit/miss counter
```

Comment:
see e.g. https://github.com/PostHog/posthog/blob/master/posthog/models/feature_flag/flag_matching.py#L846

This will come when I wire up the Rust prometheus client to this service.